File: examples/middleware/modify_response/modify_response.py
#!/usr/bin/env python
import sys
import json

def main():
    data = sys.stdin.readlines()
    payload = data[0]

    payload_dict = json.loads(payload)
    payload_dict['response']['body'] = "body was replaced by middleware\n"

    print(json.dumps(payload_dict))

if __name__ == "__main__":
    main()

File: examples/middleware/modify_response/modify_response.js
#!/usr/bin/env node

process.stdin.resume();  
process.stdin.setEncoding('utf8');  
process.stdin.on('data', function(data) {
  var parsed_json = JSON.parse(data);

  parsed_json.response.body = "body was replaced by middleware\n";

  process.stdout.write(JSON.stringify(parsed_json));
});
